cluster/cluster.sh

```diff
@@ -49,5 +49,13 @@ function cluster::path() {
 }
 
 function cluster::kubeconfig() {
-    echo -n ${CLUSTER_PATH}/_ci-configs/${KUBEVIRT_PROVIDER}/.kubeconfig
+
+    if [ ${KUBEVIRT_PROVIDER} != "external" ]; then
+      echo -n ${CLUSTER_PATH}/_ci-configs/${KUBEVIRT_PROVIDER}/.kubeconfig
+    fi
+    if [ -n ${KUBECONFIG} ]; then
+      echo -n ${KUBECONFIG}
```

Comment:
why not always just `echo -n ${KUBECONFIG}`
and require `KUBECONFIG` to be set no matter if its internal or external
thats what we are doing on kubevirt, and this is better imho

Reply:
OK. I don't mind. How would you assert on it?
 this is the direction ?
```
if [ -n ${KUBECONFIG} ]; then
      echo -n ${KUBECONFIG}
    else
      exit 1
```

Reply:
yep or 
```
[[ -n $KUBECONFIG ]] || (echo missing ... ; exit 1)
echo -n ${KUBECONFIG}
```

Reply:
Nice. Thanks.

Reply:
[Done](https://github.com/kubevirt/cluster-network-addons-operator/compare/fd1feb96a75c8b97e7f11975c86f1d7218f840b5..33516fee265d3ce0b0cb7281d8eb308426aa518e)